lib/conductor.js

```diff
@@ -82,11 +82,22 @@ module.exports = (settings, args) => {
     return { ...service };
   };
 
+  const getRunCommands = (service, i, all) => {
+    const services = keyBy(all, 'name');
+    const local = dotenv.parse(fs.readFileSync(path.resolve(process.cwd(), args.env)));
+    const configured = settings.env;
+    if (service.run) {
+      service.run = mustache.render(service.run, { env: Object.assign(local, configured), services });
```

Comment:
At this point it might make more sense to just run the entire `conductor.json` through mustache to replace any/all vars before we do the conversion to `docker-compose.yml`?